jetty-core/jetty-deploy/src/main/config/modules/deployment-scanner.mod

```diff
@@ -20,15 +20,16 @@ etc/jetty-deployment-scanner.xml
 
 [ini-template]
 #tag::documentation[]
-## The web application deploy directory name (relative to $JETTY_BASE)
+## The web application deploy directory name, or a comma-separated
+## list of directories (relative to $JETTY_BASE).
```

Comment:
Is this true? Must they be relative to `$JETTY_BASE`? The javadoc on `XmlConfiguration.csvSplitAndResolvePaths` says it accepts relative or absolute paths.

Reply:
I clarified the module comment, in this file as well as in other files.